waltz-ng/client/roadmap/components/roadmap-scenario-diagram/roadmap-scenario-diagram.js

```diff
@@ -384,6 +410,24 @@ function controller($q,
     vm.onCloseDialog = () => {
         vm.dialog = null;
     };
+
+    vm.unhideAxis = (axis) => {
+        _.remove(vm.hiddenAxes, (d => d.id === axis.id));
+        vm.vizData = prepareData(
+            vm.scenarioDefn,
+            vm.applications,
+            vm.measurables,
+            vm.hiddenAxes);
+    };
+
+    vm.unhideAllAxes = () => {
+        vm.hiddenAxes = [];
+        vm.vizData = prepareData(
```

Comment:
PrepareData(.....) Looks like it could be added as a convenience method as all it's args are in 'vm'